tx_check.go

```diff
@@ -0,0 +1,202 @@
+package bbolt
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+// Check performs several consistency checks on the database for this transaction.
+// An error is returned if any inconsistency is found.
+//
+// It can be safely run concurrently on a writable transaction. However, this
+// incurs a high cost for large databases and databases with a lot of subbuckets
+// because of caching. This overhead can be removed if running on a read-only
+// transaction, however, it is not safe to execute other writer transactions at
+// the same time.
+func (tx *Tx) Check(keyValueStringer KeyValueStringer) <-chan error {
```

Comment:
nit: can we just rename `keyValueStringer` to `kvStringer` so as to simplify it?
```suggestion
func (tx *Tx) Check(kvStringer KeyValueStringer) <-chan error {
```

Reply:
Done